src/state/get-droppable-over.js

```diff
@@ -1,22 +1,65 @@
 // @flow
-import { isPointWithin } from './is-within-visible-bounds-of-droppable';
+import { isPointWithinDroppable } from './is-within-visible-bounds-of-droppable';
+import { patch } from './position';
+import getDraggablesInsideDroppable from './get-draggables-inside-droppable';
 import type {
   DroppableId,
   Position,
+  DraggableDimension,
+  DraggableDimensionMap,
   DroppableDimensionMap,
   DroppableDimension,
 } from '../types';
 
-export default (
+const noPadding: Position = { x: 0, y: 0 };
+
+type Args = {
   target: Position,
+  draggable: DraggableDimension,
+  draggables: DraggableDimensionMap,
   droppables: DroppableDimensionMap,
-): ?DroppableId => {
+  previousDroppableOver: ?DroppableId,
+};
+
+export default ({
+  target,
+  draggable,
+  draggables,
+  droppables,
+  previousDroppableOver,
+}: Args): ?DroppableId => {
   const maybe: ?DroppableDimension =
     Object.keys(droppables)
       .map((id: DroppableId): DroppableDimension => droppables[id])
-      .find((droppable: DroppableDimension): boolean => (
-        isPointWithin(droppable)(target)
-      ));
+      .find((droppable: DroppableDimension): boolean => {
+        const placeholderPadding = (() => {
```

Comment:
Can you add the flow type to this variable as well as to the others in this file?

`placeholderBuffer: Position`